python/grass/jupyter/setup.funcs.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import grass.script as gs
+import grass.script.setup as gsetup
+
+
+def _set_notebook_defaults():
+    """
+    This function sets several GRASS environment variables that are
+    important for GRASS to run smoothly in Jupyter. A complete list and
+    description of environment variables can be found here:
+    https://grass.osgeo.org/grass78/manuals/variables.html
```

Comment:
The whole second sentence can be deleted. Not really relevant for the user. Next programmer will hopefully be able to find that page without specific link. The link itself is adding complexity, it contains `78`, so it needs to be changed for every minor version - in fact it is not matching this version. Alternative would be to link `variables.html` in the source code, but I don't think that's needed in the first place.